coretex/cli/modules/cron.py

```diff
@@ -36,15 +37,12 @@ def jobExists(script: str) -> bool:
     return any(line.endswith(script) for line in existingLines)
 
 
-def scheduleJob(configDir: Path, script: str) -> None:
-    if jobExists(script):
-        return
-
+def scheduleJob(scriptName: str) -> None:
     existingLines = getExisting()
-    cronEntry = f"*/30 * * * * {configDir / script}\n"
+    cronEntry = f"*/30 * * * * {DEFAULT_VENV_PATH}/bin/python {RESOURCES_DIR}/{scriptName} >> {DEFAULT_VENV_PATH.parent}/out.txt  2>&1\n"
```

Comment:
Why don't we make it so that it sources the environment and then runs "coretex node update" command directly and set a flag for it to reject all prompts like the one we have where when the command is executed if the Node is busy and it prompts user if they want to stop the Node to update it. We can add a flag which automatically anwers "No" to these questions.